docs/usage.md

```diff
@@ -0,0 +1,33 @@
+# Usage
+
+## Available commands
+
+### Get list of users
+
+:::note
+This command is only available for console super administrator
+:::
+
+```bash
+$ miactl get users
+```
+
+### Create new user
+
+:::note
+This command is only available for console super administrator
+:::
+
+```bash
+$ miactl add user --username="john.smith@example.com"
```

Comment:
Why `--username` option and not something like: `miactl add user john.smith@example.com`? Also for the delete

Reply:
I've changed to it and also added some new parameters